api/capturesource_test.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/ausocean/openfish/api/api"
+	"github.com/ausocean/openfish/api/ds_client"
+	"github.com/ausocean/openfish/api/handlers"
+	"github.com/gofiber/fiber/v2"
+)
+
+func TestCreateCaptureSource(t *testing.T) {
+
+	type testCases struct {
```

Comment:
You could define test cases here with an anonymous struct so that the for loop condition section doesn't get crammed.
`
cases := []struct{
    req string 
    expect int 
}{
    ....
}

`

Reply:
Good suggestion